### DISCUSSION TITLE
人类之光

### DISCUSSION THREAD

**Discussion by xieshaoping** [xai-org/grok-1#45] [General]
非常好，支持开源精神！

**Comment by SpaceSkater** [comment]
支持老马🫡

**Comment by tonglinfang** [comment]
老馬就是不一樣

**Comment by jonnie1688** [comment]
老马好样的

**Comment by Iamnotphage** [comment]
留名

**Comment by dreamfly11712** [comment]
老马牛批

**Comment by MiniKoro** [comment]
有人又要创新了

**Comment by SiKulou** [comment]
硬核马斯克~

**Comment by EvanWonghere** [comment]
老马还是牛的，说到做到

**Comment by tomchon** [comment]
所以国内的会抄写吗

**Reply by Hashan0217** [reply]
小商贩抄袭不了 但是中小型企业 开源的下载过来 搞个界面，说是国产自研 跟那个编辑器一样哈哈哈哈哈 都一个尿性

**Comment by cassius425** [comment]
国内又要出大模型了

**Reply by Toanlele** [reply]
看谁遥遥领先了。

**Reply by Hashan0217** [reply]
哈哈哈哈哈

**Comment by iyvan** [comment]
先留名

**Comment by JensenYang** [comment]
支持一波

**Comment by 321947428** [comment]
马总格局遥遥领先。~

**Comment by liass001** [comment]
人类之光，留名支持，见证历史

**Comment by clearcodecn** [comment]
国内马上安排跟进！

**Comment by HauUhang** [comment]
Musk🐮逼！

**Comment by beary** [comment]
终于开源了，咱们可以开始自研啦！

**Comment by stacklix** [comment]
老马威武，人类之光

**Comment by Bian-Sh** [comment]
有趣~

**Comment by qinrr** [comment]
神，地球人的骄傲

**Comment by CelestialCoder03** [comment]
不是，grok-1靠着314B的参数，然后效果跟只有1/6大的mixtral和1/4大的qwen-1.5-72b一个水平，就这也能尬吹？

**Reply by xieshaoping** [reply]
不是，大家支持的是grok-1的开源的思想，而不是吹它的技术，这你也能尬喷？

**Reply by CelestialCoder03** [reply]
开源的多了去了，怎么马斯克这个就是人类之光就是神？你怎么不吹meta，人家开源了llama，llama2，马上还开源llama3

**Reply by xieshaoping** [reply]
找出100种理由，你能找出101种理由反驳，不跟喷子论对错

**Reply by Tracalight** [reply]
> 开源的多了去了，怎么马斯克这个就是人类之光就是神？你怎么不吹meta，人家开源了llama，llama2，马上还开源llama3

你猜是什么人会来这里排队喊牛比，心智都没成熟看着营销号视频问着味就来了

**Reply by xieshaoping** [reply]
我不敢苟同。我个人认为这个意大利面就应该拌42号混凝土。

**Reply by 906051999** [reply]
营销的王

**Reply by loadingyy** [reply]
推理速度是80B大模型的速度，而且开源的代码比LLama2代码的细节优化太多了，最重要的是，训这种大模型成本是很高的，开源值得支持

**Reply by JasonQ1123** [reply]
> 推理速度是80B大模型的速度，而且开源的代码比LLama2代码的细节优化太多了，最重要的是，训这种大模型成本是很高的，开源值得支持

开源的大模型多了去了，况且这个模型的推理速度很快吗？自己看看discuss里面唯一跑起来的那个才7token per ish，慢的要死还一堆问题，麻烦吹捧之前先好好的自己研究一下好吧？不要见到你们的源神就一顿狂吹，开源是值得支持但是不值得吹捧你们🐴爹的臭脚

**Reply by loadingyy** [reply]
笑嘻了，我是没环境跑，我也不用看discuss，我看源码就够了，学到了很多东西

**Comment by supersal3478** [comment]
i don't understand anything, nice to meet you all guys LOL!! :D

**Reply by bluevisor** [reply]
we are just admiring Elon Musk...

**Reply by supersal3478** [reply]
LOLs kk yeah he is a piece of art! hehe

**Reply by supersal3478** [reply]
thank you @bluevisor

**Reply by Fodesu** [reply]
LoL

**Comment by xiaozhang-cn** [comment]
老马之前怒喷open ai我还觉得他魔怔, 现在老马真把自己家的牛捐出来了,真牛. 他要真上了火星真是21世纪标志性人物了

**Reply by Okkband** [reply]
性能这么差，还好意思放出来。

**Comment by sunmoon919** [comment]
牛逼！老马才是真格局，人之大道，什么地缘政治，无非都是资本家玩的手段。

**Comment by gotoeasy** [comment]
不用grok会显得没格局，要的是这种格局

**Comment by zhengyu111** [comment]
老马这种言行一致的人，工程师精神，开源精神，人类之光

**Comment by saksim** [comment]
曾经的我质疑老马，现在的我觉得老马就是知行合一的人。 可以，我服

**Comment by Stargate007** [comment]
马斯克，人类之光，留名支持。

**Reply by JasonQ1123** [reply]
打卡纯纯shabi玩意😓😓😓

**Comment by wylsnake** [comment]
If you dare to open source, I will develop it myself

**Comment by liuyajun74** [comment]
The Great Musk

**Comment by rockiecxh** [comment]
为开源精神点赞！

**Comment by Whalekv** [comment]
此言为见证历史而留

**Comment by jt-z** [comment]
elon mask ！

**Comment by sherwoodrider** [comment]
马斯克格局真的大，一己之力推动国内科技实现抄越

**Comment by 706697916** [comment]
感谢老马，开源伟大！

**Comment by NoodlesGentleman** [comment]
elon mask ！

**Comment by zhouguozhao** [comment]
合照

**Comment by ywcmoon** [comment]
牛逼！

**Comment by ahern88** [comment]
老马真心牛逼，不得不服，不愧是我偶像

**Comment by Toanlele** [comment]
你这么牛逼，要不你自己写一个开源噻











。

---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年3月19日(周二) 上午9:15
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [xai-org/grok-1] 人类之光 (Discussion #45)




 
性能这么差，还好意思放出来。

**Comment by CircleZy** [comment]
留下历史 人类之光

**Comment by Nextheartbeat** [comment]
牛逼!

**Comment by ZhangJianBeiJing** [comment]
开源精神！

**Comment by zard-zhang** [comment]
国内厂商：“我们取得了重大突破！”

**Reply by Bian-Sh** [reply]
你又知道了？

**Comment by huaixiaozi110** [comment]
怎么下载学习学习，虽然没有办法跑起来 哈哈

**Comment by Flynn-X** [comment]
2024-3-19 19:24 还没见人跑起来

**Comment by sakisaki01** [comment]
源神

**Reply by JasonQ1123** [reply]
懂不懂代码界源神的含金量！

**Comment by ProjectTorino** [comment]
> 非常好，支持开源精神！

源神，启动

**Comment by chaoqiangl** [comment]
他妈地球之光

**Comment by TomSjtu** [comment]
支持马老师
